openapi.yaml

```diff
@@ -998,19 +998,19 @@ parameters:
     in: path
     name: pizzaId
     required: true
-    type: string
+    type: integer
     description: ID of pizza to retrieve
   doughId:
     in: path
     name: doughId
     required: true
-    type: string
+    type: integer
```

Comment:
changing this and others to `integer` because our database column uses an integer and it simplifies the code a lot if we don't have to do the conversion ourselves

Reply:
Based on the JSON API, ID *MUST* be a string: https://jsonapi.org/format/#document-resource-object-identification

Reply:
Hmm, I see why we had it that way originally then. It doesn't seem like other learning project APIs handle the case where a user passes in a string instead of an int, is that acceptable behavior here?

Reply:
Which one? Looks like Parks API, Poker API and Game Curator API are all treating IDs as strings?

Reply:
Sorry, meant to say it doesn't look like any of them handle the case where someone passes a string that can't be parsed to an integer, like `'abc'` or similar as opposed to `'1'` or `'2'`.

Reply:
Actually they did. They added [pattern](https://github.com/osu-mist/game-curator-api/blob/master/openapi.yaml#L94) in openapi file so that it will be validated automatically by the package we are using.

Reply:
Oh, perfect! I'll go with that solution then.